packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -27,6 +29,20 @@ type DropdownProps = {|
      */
     items: Array<DropdownItem>,
 
+    /**
+     * An optional handler to set the searchText of the parent. When this and
+     * the searchText are provided, SearchTextInput will be displayed at the
+     * top of the dropdown body.
+     */
+    onSearchTextChanged?: (searchText: string) => mixed,
```

Comment:
Nice with the `=> mixed`. Makes using these props much easier.

To that end, do we ever envisage a case where we want to optionally set this, like:

```
<Component
   onSearchTextChanged={this.someFlag ? this.handler : null}
/>
```

What do you think about typing this as:

`onSearchTextChanged?: ?((searchText:string) => mixed)`

So that it enables that inline syntax.

Reply:
/cc @kevinbarabash @jandrade